src/helics/core/CommonCore.cpp

```diff
@@ -2426,15 +2426,15 @@ void CommonCore::setIdentifier(std::string_view name)
     }
 }
 
-static const std::map<std::string_view, std::pair<std::uint16_t, bool>> mapIndex{
-    {"global_time", {CURRENT_TIME_MAP, true}},
-    {"global_status", {GLOBAL_STATUS, false}},
-    {"dependency_graph", {DEPENDENCY_GRAPH, false}},
-    {"data_flow_graph", {DATA_FLOW_GRAPH, false}},
-    {"global_state", {GLOBAL_STATE, true}},
-    {"global_time_debugging", {GLOBAL_TIME_DEBUGGING, true}},
-    {"global_flush", {GLOBAL_FLUSH, true}},
-};
+static const std::map<std::string_view, std::pair<std::uint16_t, QueryReuse>> mapIndex{
+    {"global_time", {CURRENT_TIME_MAP, QueryReuse::DISABLED}},
+    {"global_status", {GLOBAL_STATUS, QueryReuse::DISABLED}},
+    {"dependency_graph", {DEPENDENCY_GRAPH, QueryReuse::ENABLED}},
+    {"data_flow_graph", {DATA_FLOW_GRAPH, QueryReuse::ENABLED}},
+    {"barriers", {BARRIERS, QueryReuse::DISABLED}},
+    {"global_state", {GLOBAL_STATE, QueryReuse::DISABLED}},
+    {"global_time_debugging", {GLOBAL_TIME_DEBUGGING, QueryReuse::DISABLED}},
+    {"global_flush", {GLOBAL_FLUSH, QueryReuse::DISABLED}}};
```

Comment:
It seems like the QueryReuse value mapping of the std::pair is different after the change from a bool to QueryReuse -- is that intentional?

Specifically, if true -> QueryReuse::DISABLED and false -> queryReuse::Enabled, then I'd expect global_status to be QueryReuse::ENABLED.

Reply:
Yes,  I realized that the previous value for global_status was not correct, that query can change and should be reset.    So was probably a bug.